### DISCUSSION TITLE
Webgui Problem

### DISCUSSION THREAD

**Discussion by gnome4711** [jomjol/AI-on-the-edge-device#1314] [General]
### The Problem

After completion of the setup the webgui shows the following.
I have also one installed on a gesmeter with runs fine (but different firmware). 

Is this a known issue? Can I fix this without reflashing/configuring?

![grafik](https://user-images.githubusercontent.com/7687563/200921504-3d83c90c-4460-42ec-bef0-6031beeaa3a7.png)

**Comment by caco3** [comment]
press F5 ro reload the page.
The webbrowser did not load allneeded data.

And use the latest version.

**Comment by gnome4711** [comment]
I use the latest version
f5? (seriously??) don't get me wrong but don't you think I tried this many times?? btw: doesn't change anything
But thanks!!

**Reply by caco3** [reply]
> I use the latest version 

No you are not! The screenshot clearly shows `11.2.0`, but the latest release is on `12.0.1`!

> f5? (seriously??)

Yes!
Most of the time the browser just failed to load the stylesheet. F5 usually fixes this.

if not - as @jomjol stated, the SD-Card misses some files or is corrupted => do an OTA or create the card again.

**Reply by gnome4711** [reply]
I strongly asumed downloading this way gives me the latest!

![grafik](https://user-images.githubusercontent.com/7687563/201374893-e7acf350-547a-4318-a9f9-0474d81245f5.png)

**Reply by caco3** [reply]
use the pre-built release (right side, 12.0.1)

**Comment by jomjol** [comment]
Then most probalby something gone wrong during the firmware update or a file on the sd-card is broken. You could try an update again ("OTA Update" link still available) or take out the sd-card and copy the html directory manually-
